tox.ini

```diff
@@ -19,11 +19,11 @@ commands = flake8 {posargs} src/github3/ tests/unit/ tests/integration/
 
 [testenv:build]
 skip_install = true
-basepython = python3.10
+basepython = python3.11
 deps =
-    build
+    hatch
 commands =
-    python -m build
```

Comment:
Please use the standard build interface, not hatch

Reply:
@sigmavirus24 if you try it will break. Not sure if we can fix given the bug in https://github.com/pypi/warehouse/issues/10030

Reply:
Oh, right. I forgot that PyPI are at war with PyPA standards, sigh.

Reply:
With `build -s` it works.. thanks @offbyone for the hint.